addon/components/ember-th/template.hbs

```diff
@@ -2,21 +2,8 @@
   {{yield columnValue columnMeta}}
 {{else}}
   {{columnValue.name}}
-{{/if}}
-
-{{#if isSorted}}
```

Comment:
The corresponding properties on the ember-th component instance should probably be dropped?

For example: https://github.com/Addepar/ember-table/blob/master/addon/components/ember-th/component.js#L78

Reply:
done